### PR TITLE
Install CompatHelper as GitHub Action

### PR DESCRIPTION
**Pull request by sethaxen**

### CODE REVIEW

File: .github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: CompatHelper
+
+on:
+  schedule:
+    - cron: '00 00 * * *'
```
Comment (devmotion) [inline .github/workflows/CompatHelper.yml]: Can we change the frequency to once a day? IMO it's not necessary to run it every hour.

File: .github/workflows/CompatHelper.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: 1.3
```
Comment (devmotion) [inline .github/workflows/CompatHelper.yml]: Maybe use version 1.4?
```suggestion
          version: 1.4
```

File: .github/workflows/CompatHelper.yml (hunk 1 above)
Comment (devmotion) [reply .github/workflows/CompatHelper.yml]: Oh sorry, it seems it's already once a day... I'm always confused by the cron notation :smile:

Comment (sethaxen) [reply .github/workflows/CompatHelper.yml]: No worries, yeah I just used the boilerplate from CompatHelper's README. Been a while since I touched cron.

**Pull request merged**